docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,651 @@
+---
+sidebar_label: jira
+description: Integrates with Jira to manage issues, comments, and track promotion workflows.
+---
+
+# `jira`
+
+:::info Enterprise Feature
+This promotion step is only available in Akuity Promote, you can use this feature in Kargo v1.6+ offered on [Akuity Platform](https://akuity.io/akuity-platform).
+:::
+
+The `jira` promotion step provides comprehensive integration with Jira, allowing you to create, update, delete, and search for issues, manage comments, and track promotion workflows. This is particularly useful for maintaining traceability between your deployment processes and project management activities.
+
+This promotion step supports various operations including issue management, comment handling, and status tracking, making it a powerful tool for promotion workflows that require coordination with project management systems.
+
+## Credentials Configuration
+
+All Jira operations require proper authentication credentials stored in a Kubernetes `Secret`.
+
+| Name                     | Type     | Required | Description                                                                                                                  |
+| ------------------------ | -------- | -------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `credentials.secretName` | `string` | Y        | Name of the `Secret` containing the Jira credentials.                                                                        |
+| `credentials.namespace`  | `string` | N        | Namespace for the credentials `Secret`. Can be project or cluster secret namespace. If empty, the project namespace is used. |
```

Comment:
```suggestion
| `credentials.namespace`  | `string` | N        | Namespace containing the credentials `Secret`. This can be the Project's namespace or the designated cluster secrets namespace. If empty, the Project namespace is assumed. |
```

Reply:
I can't remember where we landed on it, but if the cluster secrets namespace isn't configurable in Akuity Promote and this feature is exclusive to Akuity Promote, should we say here what the name of the the cluster secrets namespace actually is?

Reply:
Actually... another concern popped into my head. Other features using the "cluster secrets namespace" use that namespace as a repository for Secrets referenced by, as the name implies, _cluster-scoped things._

The way it's being used here seems to be quite a departure from that. It seems this is being used in a way that is more consistent with what "global credential namespaces" had been intended for -- i.e. sharing across Projects.

As it stands now, it seems referencing cluster-scoped secrets from a Project-scoped resource creates a possible vulnerability by allowing any Project to access Secrets belonging to the "Kargo admin" (my shorthand for whoever maintains cluster-scoped Kargo resources).

Reply:
Marking this as resolved and we can move the convo to [here](https://github.com/akuity/kargo/pull/4497#discussion_r2181199754).